tests/conftest.py

```diff
@@ -588,7 +588,7 @@ def mem_size_mib():
     return 256
 
 
-@pytest.fixture(params=[True, False])
+@pytest.fixture(params=[True, False], ids=["PCI", "MMIO"])
 def pci_enabled(request):
```

Comment:
shall it be renamed too? pci_enabled == MMIO doesn't read smoothly

Reply:
The problem here is that `pci_enable` is a toggle. I can rename `MMIO` to `NoPCI` if you want. Or do you have other suggestions?

Reply:
Would PCI_ON/PCI_OFF be too verbose?